.vib/kibana/cypress/cypress/integration/kibana_spec.js

```diff
@@ -6,7 +6,7 @@ before(() => {
   skipTheWelcomeScreen();
 });
 
-it('allows uploading a file', () => {
+it('allows uploading data', () => {
   cy.get('a[data-test-subj="uploadFile"]').click();
```

Comment:
These types of selectors tend to be pretty much accurate, is the element type really needed here?
```suggestion
  cy.get('[data-test-subj="uploadFile"]').click();
```

Reply:
Agree

Reply:
Modified every affected check. Thanks!